### ISSUE TITLE
setPermanentCookieJson is deprecated, but why?

### ISSUE THREAD

**Issue by phadej** [obsidiansystems/rhyolite#66]
deprecation message says to use `defaultCookieJson`, but it seems that you'd
reimplement `setPermanentCookieJson` then, as `defaultCookieJson` doesn't set anything?

```haskell
{-# DEPRECATED setPermanentCookieJson "Use defaultCookieJson instead" #-}
setPermanentCookieJson :: (MonadJSM m, HasJSContext m, ToJSON v) => DOM.Document -> Text -> Maybe v -> m ()
setPermanentCookieJson d k = setPermanentCookie d <=< defaultCookieJson k
```

I think that the deprecation should be removed, or clarified.

**Comment by 3noch** [comment]
My guess is that
```haskell
setPermanentCookie d <=< defaultCookieJson k
--vs
setPermanentCookieJson d k
```
didn't justify it's seat in the namespace since the implementation is hardly longer than the name.

**Comment by 3noch** [comment]
But you're right that the deprecation message is confusing.

**Comment by phadej** [comment]
the `setPermanentCookieJson` is however quite handy, and it's natural "inverse" of `getCookieJson`.

**Comment by ali-abrar** [comment]
> the setPermanentCookieJson is however quite handy, and it's natural
"inverse" of getCookieJson.

I agree

**Comment by ali-abrar** [comment]
Fixed via 997c4e210305cca29035f11d4a9806887f62d11a
